### PR TITLE
Cockpit 208

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-208.html

### CODE REVIEW

File: _posts/2019-11-27-cockpit-208.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
+
+Cockpit no longer offers the confusing "Default" choice when mounting a filesystem. Now every new filesystem requires the specification of a mount point. Additionally, Cockpit hides the distinction between configuration (`/etc/fstab`) and run-time state (`/proc/mounts`). Changes made in Cockpit always apply to both the configuration and run-time state. When configuration and run-time state differ from each other, Cockpit will warn about that and will let you bring them back in sync.
```
Comment (allisonkarlitskaya) [inline _posts/2019-11-27-cockpit-208.md]: I'm not sure what you're talking about here: the start of the sentence suggests that we will ignore the "mount point" flag stored in the filesystem itself, but then you say "every new filesystem requires the specification of a mount point".

What does this mean?  Do you mean that when creating a filesystem (ie: mke2fs) then mountpoint is required or when mounting a filesystem, mountpoint is required?

File: _posts/2019-11-27-cockpit-208.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+![Storage format dialog](/images/storage-format-mountpoint.png)
+
+### Machines: Support transient virtual networks and storage pools
+
+Transient resources are resources which exist only while they are running and cease to exists once they stop running/are deactivated. Cockpit now disables or hides functionality like autostart or deletion for transient resources, as these would fail:
```
Comment (allisonkarlitskaya) [inline _posts/2019-11-27-cockpit-208.md]: This reads like the resource itself is running.  Is it rather the associated machine is running?

Comment (allisonkarlitskaya) [inline _posts/2019-11-27-cockpit-208.md]: Use of a colon at the end here is a bit inconsistent.

File: _posts/2019-11-27-cockpit-208.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![Transient virtual network](/images/machines-transient-network.png)
+
+### Machines: A size sliders for VM creation
+
+The "Create New Virtual Machine" dialog now has sliders to configure the memory and disk sizes:
```
Comment (allisonkarlitskaya) [inline _posts/2019-11-27-cockpit-208.md]: Colon here as well.

File: _posts/2019-11-27-cockpit-208.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+![Machine creation size sliders](/images/machines-size-sliders.png)
+
+### Logs: Use reportd for reporting crashes
+
+Cockpit previously sent problem/crash reports directly to [ABRT Analytics](https://github.com/abrt/faf) when reporting them on the Logs page. That got changed to using the new "reportd" framework. This supports reporting to more places, such as [Bugzilla](https://bugzilla.redhat.com/), and handles authentication in a better way.
```
Comment (allisonkarlitskaya) [inline _posts/2019-11-27-cockpit-208.md]: "got changed" is probably too informal for release notes.  Maybe something like "Cockpit now uses the reportd framework" or so?

File: _posts/2019-11-27-cockpit-208.md (hunk 2 above)
Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: I updated the description.

Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: Right, replaced with a period.

File: _posts/2019-11-27-cockpit-208.md (hunk 3 above)
Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: Fixed.

File: _posts/2019-11-27-cockpit-208.md (hunk 4 above)
Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: Thanks, fixed.

File: _posts/2019-11-27-cockpit-208.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: It's the former, on fs creation you now always need to specify a mount point. I changed it a bit, is it clearer now? (I copied that from the PR from @mvollmer, maybe you also have a suggestion?)

File: _posts/2019-11-27-cockpit-208.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+![Storage format dialog](/images/storage-format-mountpoint.png)
+
+### Machines: Support transient virtual networks and storage pools
+
+Transient resources exist only while they are active and cease to exists once they get deactivated. Cockpit now disables or hides functionality like autostart or deletion for transient networks and storage pools.
+
+![Transient virtual network](/images/machines-transient-network.png)
+
+### Machines: A size sliders for VM creation
```
Comment (garrett) [inline _posts/2019-11-27-cockpit-208.md]:
```suggestion
### Machines: Sliders for disk size and memory in VM creation
```

File: _posts/2019-11-27-cockpit-208.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+The "Create New Virtual Machine" dialog now has sliders to configure the memory and disk sizes.
```
Comment (garrett) [inline _posts/2019-11-27-cockpit-208.md]: I think there is also for the memory slider in the Import VM dialog too.

File: _posts/2019-11-27-cockpit-208.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![Machine creation size sliders](/images/machines-size-sliders.png)
+
+### Logs: Use reportd for reporting crashes
```
Comment (garrett) [inline _posts/2019-11-27-cockpit-208.md]:
```suggestion
### Logs: Improve crash reporting
```

File: _posts/2019-11-27-cockpit-208.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+tags: cockpit linux
+slug: cockpit-208
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
```
Comment (garrett) [inline _posts/2019-11-27-cockpit-208.md]: I'm not sure about this change. But what if  we focus on unifying fstab and mtab?

```suggestion
### Storage: Unify saved and current mounts
```

Comment (martinpitt) [reply _posts/2019-11-27-cockpit-208.md]: The "drop default mount point" was the main change, and the static/runtime disparity more of a by-product. The main result from usability review was that few people understand this magic "default mount point" concept.

**Comment by martinpitt** [comment]
Thanks @garrett for your fixups! I squashed them.

**Comment by martinpitt** [comment]
I pushed the bodhi links, release is done now.

**Pull request merged**